test/JDBC/expected/bbf_role_admin_restrictions.out

```diff
@@ -118,7 +118,7 @@ SET SESSION AUTHORIZATION bbf_role_admin;
 GO
 ~~ERROR (Code: 0)~~
 
-~~ERROR (Message: ERROR: permission denied to set session authorization
+~~ERROR (Message: ERROR: permission denied to set session authorization "bbf_role_admin"
```

Comment:
So this change must be needed in gitfarm as well so let's keep this change separate from the rest of the changes by creating separate commits for each. The other change is being cherry-picked from gitfarm so it's anyways better to keep it separate.

Reply:
@rishabhtanwar29 Right, for the test fix, it's already a separate commit in this PR, let's not squash merge this.